source/draw/Color.ooc

```diff
@@ -176,38 +176,72 @@ ColorBgra: cover {
+	rgbToYuv: static func (color: ColorRgb) -> ColorYuv {
+		ColorYuv new(
+			((This _rgbToYuv0[color red] + This _rgbToYuv0[256 + color green] + This _rgbToYuv0[512 + color blue]) >> 8) clamp (0, 255) as Byte,
+			(((This _rgbToYuv1[color red] + This _rgbToYuv1[256 + color green] + This _rgbToYuv1[512 + color blue]) >> 8) + 128) clamp(0, 255) as Byte,
+			(((This _rgbToYuv2[color red] + This _rgbToYuv2[256 + color green] + This _rgbToYuv2[512 + color blue]) >> 8) + 128) clamp(0, 255) as Byte
+		)
+	}
+	cleanUp: static func {
```

Comment:
I agree about the global cleanup function. Ideally it would free all the static string literals too.